.github/workflows/health_base.yaml

```diff
@@ -118,7 +117,7 @@ jobs:
         if: ${{ '$action_state' == 1 }}
 
       - name: Upload coverage to Coveralls
-        if: ${{ inputs.upload_coverage }}
+        if: ${{ inputs.upload_coverage && inputs.check == 'coverage' }}
```

Comment:
This is a bugfix, before, coverage was always uploaded as `upload_coverage` defaults to `true`.